augly/video/helpers/intensity.py

```diff
@@ -3,6 +3,7 @@
 
 from typing import Any, Dict, Optional, Tuple
 
+import augly.audio.intensity as aud_intensity
```

Comment:
```suggestion
```

Don't need this anymore since we won't be calling any audio intensity functions here.